server/graphql/v2/interface/HasMembers.js

```diff
@@ -45,6 +48,20 @@ export const HasMembersFields = {
         };
       }
 
+      // Inherit Accountants and Admin from parent collective for Events and Projects
+      if ([CollectiveTypes.EVENT, CollectiveTypes.PROJECT].includes(collective.type)) {
+        const inheritedRoles = [MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER];
+        where = {
+          [Op.or]: [
+            where,
+            {
+              CollectiveId: collective.ParentCollectiveId,
+              role: { [Op.in]: args.role ? intersection(args.role, inheritedRoles) : inheritedRoles },
```

Comment:
Won't this crash if I request only `MemberRoles.BACKER`? Sounds like it could generate something like:

```sql
OR role IN ()
```

Which is invalid SQL

Reply:
Sequelize will translate this to `"Member"."role" IN (NULL)` which is valid.